components/nav/Nav.tsx

```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
 import Logo from '@/components/assets/logo.svg';
 import ProfileDropdown from '@/components/nav/ProfileDropdown';
 import Link from 'next/link';
+import { headers } from 'next/headers';
+
+export default async function NavBar() {
+  const headersList = headers();
+  const referer = headersList.get('referer');
+  let url = '/private-cloud/products/all';
+  if (typeof referer === 'string') {
+    if (referer.split('/')[3] === 'public-cloud') {
+      url = '/public-cloud/products/all';
+    }
+    if (referer.split('/')[3] === 'private-cloud') {
```

Comment:
let's use `else if`.

Reply:
doesn't work properly with `else if`

Reply:
hm.. not sure why though. can you explain little more?

Reply:
also changed, thank you